DigitalExcellence/dex-frontend
# 336 - Projects-with-complex-description-load-slow

## Description

If a project has a really long description it loads really slow, this pull request will replace the quill-view with a div that will inject the html directly into the element. This will improve the render time significantly 

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Create a project with this github link: https://github.com/quozd/awesome-dotnet
2. The project should load in fast

## #336 

Closes: #336
